pkg/apis/extensions/types.go

```diff
@@ -288,9 +278,8 @@ type RollbackConfig struct {
 const (
 	// DefaultDeploymentUniqueLabelKey is the default key of the selector that is added
 	// to existing RCs (and label key that is added to its pods) to prevent the existing RCs
-	// to select new pods (and old pods being select by new RC). See DeploymentSpec's UniqueLabelKey
-	// field for more information.
-	DefaultDeploymentUniqueLabelKey string = "deployment.kubernetes.io/podTemplateHash"
+	// to select new pods (and old pods being select by new RC).
+	DefaultDeploymentUniqueLabelKey string = "pod-template-hash"
```

Comment:
Discussed with @bgrant0607 about this, and we renamed it to be without namespace because it's too long whenever we display resources with their labels, and putting namespace is more suitable for annotations because annotations targets machines, not humans.